src/more_ops.rs

```diff
@@ -71,6 +72,228 @@ fn limbs_for_int(v: &Number) -> u32 {
     ((v.bits() + 7) >> 3) as u32
 }
 
+fn u32_from_u8(mut buf: &[u8]) -> Option<u32> {
```

Comment:
> fn u32_from_u8

I'm surprised rust doesn't have this in their stdlib.

Reply:
well, perhaps there's something similar. this has specific overflow behavior though